src/main/java/com/example/bookstoreapp/controller/BookController.java

```diff
@@ -41,4 +45,27 @@ public ResponseEntity<BookDto> getBookById(@PathVariable Long id) {
             return ResponseEntity.notFound().build();
         }
     }
+
+    @PutMapping("/{id}")
+    public ResponseEntity<BookDto> updateBook(@PathVariable Long id,
+                                              @RequestBody CreateBookRequestDto updatedBookDto) {
+        BookDto book = bookService.update(id, updatedBookDto);
+        if (book != null) {
+            return ResponseEntity.ok(book);
+        } else {
+            return ResponseEntity.notFound().build();
+        }
```

Comment:
It is better to `return ResponseEntity.ok(bookService.update(id, updatedBookDto));`